public/react/style.ts

```diff
@@ -0,0 +1,24 @@
+export const dashboardSidebarWidth = 160;
```

Comment:
Similar to above comment - should this be '160px'?

Reply:
I'd tend to keep exported dimension as numbers rather than strings so they can be used in calculations easily. If the units need to be included for clarity, we could do that where the values are used eg  ```fontSize:`${dashboardSidebarWidth/4}px```